### PR TITLE
Support the usage of mod_proxy_fcgi for rgw tests

### PR DESCRIPTION
**Pull request by andrewschoen**
This is for, http://tracker.ceph.com/issues/10551

I don't think we should merge this until http://tracker.ceph.com/issues/10808 and http://tracker.ceph.com/issues/11022 is complete, but I wanted to post it for early reviews.

This is testing out pretty well in the labs, http://pulpito.front.sepia.ceph.com/ubuntu-2015-03-10_09:19:29-teuthology-master---basic-multi/.  The ubuntu 12.04 failure is expected for fcgi because that distro is running Apache 2.2.  The rhel 7 failure was an environment issue, and we've seen those tests pass before for rhel 7.

@zmc please take a look at the additions to the teuthology suite.  I'm not sure if that's the correct place to put the rgw stuff, but I'd like to start storing simple config files that we can use when testing new functionality.  I used those same yaml files in the tasks directory (with minor changes) for manual testing as well.

### CODE REVIEW

File: suites/teuthology/rgw/tasks/s3tests-fastcgi.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+tasks:
+- install:
+    branch: firefly
+- ceph:
+- rgw: [client.0]
+- s3tests:
+    client.0:
+      rgw_server: client.0
+      force-branch: wip-10551-firefly-original
```
Comment (zmc) [inline suites/teuthology/rgw/tasks/s3tests-fastcgi.yaml]: Just a reminder that we may want to remove this before merging as the branch might go away.

**Comment by andrewschoen** [comment]
@zmc this is ready for re-review and merge.

**Comment by zmc** [comment]
Looks good, squash it up

**Comment by andrewschoen** [comment]
@zmc squashed

**Pull request merged**